### DISCUSSION TITLE
Adding export as PDF option to fields such as tickets and projects

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#16845] [How to]
I see that exporting Invoices of various kinds as PDFs is an option in Yetiforce. However, my company wants to be able to export tickets and projects as PDFs. How would I implement the Export to PDF button that can be found in the invoice modules? I have already created a PDF template for both Tickets and Projects, now I just need to implement adding the option to export as a PDF.

**Comment by rskrzypczak** [comment]
All you have to do is add a PDF template for a given module and check that the 'Export to PDF' item is checked in the profile settings in that module.
The export to PDF button may be available in each module.
